packages/diagrams/lib/src/custom_scroll_view.dart

```diff
@@ -0,0 +1,166 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:diagram_capture/diagram_capture.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/scheduler.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'diagram_step.dart';
+import 'fake_drag_scroll_activity.dart';
+
+final Duration _kTotalDuration =
+    _kScrollUpDuration
+        + _kScrollPauseDuration
+        + _kScrollDownDurationPartOne
+        + _kScrollPauseDuration
+        + _kScrollDownDurationPartTwo
+        + _kScrollPauseDuration;
+const Duration _kScrollUpDuration = Duration(seconds: 1, milliseconds: 500);
+const Duration _kScrollDownDurationPartOne = Duration(milliseconds: 800);
+const Duration _kScrollDownDurationPartTwo = Duration(seconds: 1);
+const Duration _kScrollPauseDuration = Duration(milliseconds: 900);
+
+const double _kCurveAnimationFrameRate = 60.0;
+const String _customScrollView = 'custom_scroll_view';
```

Comment:
It looks like this diagram needs to be re-generated to update the latest changes.